sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/FeatureFlagConfigurationSetting.cs

```diff
@@ -356,23 +510,12 @@ private static void WriteParameterValue(Utf8JsonWriter writer, object value)
             }
         }
 
-        private void CheckValidWrite()
-        {
-            CheckValid();
-            _originalValue = null;
-        }
-
         private void CheckValid()
         {
             if (!_isValidValue)
             {
                 throw new InvalidOperationException($"The content of the {nameof(Value)} property do not represent a valid feature flag object.");
             }
         }
-
-        private void OnFiltersCollectionChange(object sender, NotifyCollectionChangedEventArgs e)
```

Comment:
Why remove validation on collection changes?

Reply:
Because it was never wired up - so the handler itself was never called.